mitodl/ocw-studio
# styling / layout changes for the resource picker

#### Pre-Flight checklist


#### What are the relevant tickets?

closes #620

#### What's this PR do?

This makes some styling and layout tweaks to the resource picker. In particular, it

- makes the modal a little bigger
- removes the 'filter' checkbox in favor of always showing the filter input
- changes images and videos to a grid-based 'gallery' layout and polishes up the list of document titles as well
- reorders tabs

See the issue for details.

#### How should this be manually tested?

Try out the resource picker. It should work as you expect, and the styling should be a little nicer than before. You should see thumbnails for youtube videos, and you should also see a tiny version of any image you upload.

#### Screenshots (if appropriate)

<img width="717" alt="Screen Shot 2021-10-04 at 3 11 00 PM" src="https://user-images.githubusercontent.com/6207644/135910165-2095e466-9569-49db-9f07-4689806a13c2.png">
<img width="743" alt="Screen Shot 2021-10-04 at 3 11 05 PM" src="https://user-images.githubusercontent.com/6207644/135910176-ed048809-1c43-496d-8336-fd9fcb9fbd3b.png">
<img width="726" alt="Screen Shot 2021-10-04 at 3 13 39 PM" src="https://user-images.githubusercontent.com/6207644/135910434-0e076922-0849-4e73-bc4c-bf317598e82d.png">
<img width="703" alt="Screen Shot 2021-10-04 at 3 13 43 PM" src="https://user-images.githubusercontent.com/6207644/135910439-73e59b35-0995-4979-ae52-a2451a75f888.png">
